### ISSUE TITLE
Remove references to MET_BASE in R scripts

### ISSUE THREAD

**Issue by georgemccabe** [dtcenter/MET#1285]
Some of the scripts in met/scripts/Rscripts require MET_BASE be set to run. We should remove the dependency so that the scripts can run without MET_BASE. A few of the scripts (like plot_tcmpr.R and plot_probri.R) also refer to MET_INSTALL_DIR. The scripts that require MET_BASE should be modified to use MET_INSTALL_DIR to determine where the other R scripts can be found. This issue came up while investigating a METplus issue (https://github.com/NCAR/METplus/issues/224) to remove MET_BASE from being set anywhere in the METplus wrappers. TCMPRPlotter calls one of the R scripts that requires MET_BASE to be set, so it will fail if MET_BASE is removed from the wrapper.

## Describe the Enhancement ##
Modify all of the files in Rscripts that reference or rely on MET_BASE. Instead use MET_INSTALL_DIR to find other scripts. The scripts should run with and without MET_BASE set in the user's environment.

### Time Estimate ###
2-4 hours including testing

### Relevant Deadlines ###
NONE

### Funding Source ###
NONE

### Assignee ###
- [X] Select **engineer(s)** or **no engineer** required (John HG)
- [X] Select **scientist(s)** or **no scientist** required (Not required)

### Labels ###
- [X] Select **component(s)**
- [X] Select **priority**
- [X] Select **requestor(s)**

### Projects and Milestone ###
- [X] Review **projects** and select relevant **Repository** and **Organization** ones
- [X] Select **milestone**

## Define Related Issue(s) ##
Consider the impact to the other METplus components.
- [X] [METplus](https://github.com/NCAR/METplus/issues/new/choose), [MET](https://github.com/NCAR/MET/issues/new/choose), [METdb](https://github.com/NCAR/METdb/issues/new/choose), [METviewer](https://github.com/NCAR/METviewer/issues/new/choose), [METexpress](https://github.com/NCAR/METexpress/issues/new/choose), [METcalcpy](https://github.com/NCAR/METcalcpy/issues/new/choose), [METplotpy](https://github.com/NCAR/METplotpy/issues/new/choose)
Impact to METplus, see: https://github.com/NCAR/METplus/issues/224

## Enhancement Checklist ##
See the [METplus Workflow](https://ncar.github.io/METplus/Contributors_Guide/github_workflow.html) for details.
- [X] Complete the issue definition above.
- [x] Fork this repository or create a branch of **develop**.
Branch name: `feature_<Issue Number>_<Description>`
- [x] Complete the development and test your changes.
- [x] Add/update unit tests: added unit_plot_tc.xml
- [X] Add/update documentation: none required
- [X] Push local changes to GitHub.
- [X] Submit a pull request to merge into **develop**, listing the **\<Issue Number\>** in the title.
- [x] Iterate until the reviewer(s) accept and merge your changes.
- [x] Delete your fork or branch.
- [x] Close this issue.

**Comment by JohnHalleyGotway** [comment]
I removed references to MET_BASE from the plot_tcmpr.R script.  I added a unit test (unit_plot_tc.xml) to exercise the plot_tcmpr.R script and added that to the list in test/bin/unit_test.sh.  In testing that, I found that R errors out when MET_INSTALL_DIR is defined with an embedded environment variable.  It doesn't automatically expand them. So I added a system call to plot_tcmpr.R and plot_probri.R to do so.

I'd initially intended to also include a call to plot_probri.R in unit_plot_tc.xml, but it didn't go well.  That plotting script either needs to be re-worked or removed from the repo.  But that can be a separate issue.
